packages/table/src/Table.ts

```diff
@@ -443,6 +446,9 @@ export class Table extends SizedMixin(SpectrumElement, {
     }
 
     protected renderVirtualizedItems(index?: number): void {
+        // Rendering updates into the table while disconnected can
+        // cause runaway event binding in ancestor elements.
```

Comment:
I'd love to hear more about this disconnected callback addition and how it adds to the robustness of the code! Is this related to the event binding we talked about before?

Reply:
This is not exactly as I expected, _but_ when you render this list while the element is disconnected, which is not something we were planning for initially, it will render a list into the ether that adds event listeners, but can never haven them removed. Remove and attach the Table a bunch of times and you end up with a lot of those!

This gets worse when you don't include the code in `disconnectedCallback()`. There we render to "nothing" which is needed to get actual event cleanup.

So now, instead of adding two extra listeners every time the Table is removed and added back, we clean up the first event listeners, don't add uncontrolled listeners and then add back the needed listers when reconnecting.